src/ethz_snow/constants.py

```diff
@@ -200,6 +231,107 @@ def calculateDerived(fpath: Optional[str] = None) -> dict:
         "beta_solution",
     ]
 
+    if config["configuration"] == "VISF":
+        if config["temperature"] == "homogeneous":
+            raise NotImplementedError(
+                (
+                    f'For simulating "{config["configuration"]}" '
+                    + "a spatial model is required."
+                )
+            )
+
+    # set up additional parameters for VISF
+    p_vac = float(config["VISF"]["p_vac"])
+    kappa = float(config["VISF"]["kappa"])
+    Dh_evaporation = float(config["VISF"]["Dh_evaporation"])
+    m_water = float(config["VISF"]["m_water"])
+    t_vac_start = float(config["VISF"]["t_vac_start"])
+    t_vac_duration = float(config["VISF"]["t_vac_duration"])
+
+    constVars.extend(
+        [
+            "p_vac",
+            "kappa",
+            "Dh_evaporation",
+            "m_water",
+            "t_vac_start",
+            "t_vac_duration",
+        ]
+    )
+
+    if config["configuration"] == "jacket":
+        if config["temperature"] != "spatial_2D":
+            raise NotImplementedError(
+                (
+                    f'For simulating "{config["configuration"]}" '
+                    + "a 2D model is required."
+                )
+            )
+
+    # set up additional parameters for VISF
```

Comment:
I assume this comment should read "jacket", not VISF?

Reply:
Correct, changed.